extra/exports

```diff
@@ -556,3 +407,221 @@ uuid_nil
 uuid_unpack
 _say
 _say_strerror
+
+# LuaJIT API {{{
+
+# Interfaces from Lua 5.1 Reference Manual (i.e. Lua C API
+# functions from lua.h).
+# XXX: Interfaces implemented via C macros are also listed below
+# only for consistency.
+lua_atpanic
+lua_call
+lua_checkstack
+lua_close
+lua_concat
```

Comment:
I agree to add `lua_getgccount` and `lua_open`: I've found them in the index [here](https://www.lua.org/ftp/refman-5.0.pdf). However, all the others look exactly like "some useful macros". I would like to not mention this explicitly here, since the main purpose of listing macros in the exports file: ensure we do not lost something from Lua reference manual.

Reply:
\*Sigh\*

I've found all of them in [Lua 5.0 sources](https://www.lua.org/source/5.0/lua.h.html) and stubs for compatibility in [Lua 5.1 sources](https://www.lua.org/source/5.1/lua.h.html). Added everything to the listing.

```diff
diff --git a/extra/exports b/extra/exports
index f1ab9e695..82c075cdf 100644
--- a/extra/exports
+++ b/extra/exports
@@ -411,6 +411,23 @@ _say_strerror
 
 # LuaJIT API {{{
 
+# Interfaces from Lua 5.0 Reference Manual (i.e. Lua C API
+# function from lua.h). Mostly deprecated or substituted by
+# something new. Left only for compatibility.
+# XXX: Interfaces implemented via C macros are also listed below
+# only for consistency.
+# <lua_getgccount> is a macro to <lua_gc>
+# <lua_getregistry> is a macro to <lua_pushvalue>
+# <lua_open> is a macro to <luaL_newstate>
+# <lua_strlen> is a macro to <lua_objlen>
+
+# Auxiliary interfaces from Lua 5.0 Reference Manual (i.e. Lua
+# auxiliary library functions from lauxlib.h). Mostly deprecated
+# or substituted by something new. Left only for compatibility.
+# XXX: Interfaces implemented via C macros are also listed below
+# only for consistency.
+# <luaL_putchar> is a macro to <luaL_addchar>
+
 # Interfaces from Lua 5.1 Reference Manual (i.e. Lua C API
 # functions from lua.h).
 # XXX: Interfaces implemented via C macros are also listed below

```

I still have some doubts regarding `lua_assert`, since it might be undefined (i.e. be no-op).